README.md

```diff
@@ -1 +1,27 @@
-# software-process
\ No newline at end of file
+# software-process
+
+## Organization
```

Comment:
We need to say something about the organization. I suggest that we write something like the following: 
Our team consists out the following eight members:
* Alex van Manen (Scrum Master)
* Pim Tegelaar (Vice Scrum Master)
* Job Witteman
* Erik Verhoofstad
* Alberto Martinez de Murga
* Ger Krijnen
* Carla Alvarado
* Joel James Bartholomew
Instead of assigning tasks. People will take a task from the Scrum Board and work on that task. When they are done  they will put it in the next state and some one else will verify it and they go on to do there next task.